plugin/src/main/kotlin/tanvd/grazi/spellcheck/GraziSpellchecker.kt

```diff
@@ -86,11 +83,11 @@ object GraziSpellchecker : GraziStateLifecycle {
                     (IdeaSpellchecker.hasProblem(word, project, language) || !Text.isLatin(word))
         }.toSet()
 
-        if (typos.size == checkers.size) {
+        if (typos.map { it.info.lang }.toSet().size == checkers.size) {
             typos.find { it.info.lang == checkerLang }?.let { typo ->
-                val fixes = mutableListOf<String>()
-                typos.flatMap(Typo::fixes).forEach { fixes.add(it) }
-                fixes.sortWith(Comparator { o1, o2 -> Text.levenshteinDistance(o1, word).compareTo(Text.levenshteinDistance(o2, word)) })
+                val fixes = typos.flatMap(Typo::fixes).sortedWith(
+                        Comparator { o1, o2 -> Text.Levenshtein.distance(o1, word).compareTo(Text.Levenshtein.distance(o2, word)) }
```

Comment:
Let's move this comparator to `Text.Levenstein` and call it accordingly :)

Reply:
Done